### PR TITLE
Document how to set-up additional data lakes for rasa-pro-analytics

### PR DESCRIPTION
**Pull request by ancalita**
**Proposed changes**:
- Completes [ATO-461](https://rasahq.atlassian.net/browse/ATO-461)

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [X] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: changelog/11721.doc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Add docs on how to set-up additional data lakes for rasa-pro analytics pipeline.
```
Comment (m-vdb) [inline changelog/11721.doc.md:1]:
```suggestion
Add docs on how to set-up additional data lakes for Rasa Pro analytics pipeline.
```

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 2]
```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
 
 :::
 
+### BigQuery
+
+These instructions assume you have running a PostgreSQL deployment in [CloudSQL](https://cloud.google.com/sql/docs/postgres).
```
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:225]:
```suggestion
These instructions assume you have a running PostgreSQL deployment in [CloudSQL](https://cloud.google.com/sql/docs/postgres).
```

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 3]
```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
...
+You can however use [Datastream for BigQuery](https://cloud.google.com/datastream/docs) to stream data from PostgreSQL
+hosted anywhere on any cloud.
+
+[Prerequisites](https://cloud.google.com/datastream/docs/before-you-begin) to using DataStream include:
+- creating a Google Cloud account where you create your Google Cloud project
+- enabling billing for the project
+- enabling the Datastream API
+- assigning Datastream Admin role to your user account
+- additional Datastream networking connectivity [requirements](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery#requirements)
```
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:234]: I don't think it makes sense to list all of these prerequisites here. But linking to them is a good idea yeah

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 4]
```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
 
 :::
 
+### BigQuery
+
+These instructions assume you have running a PostgreSQL deployment in [CloudSQL](https://cloud.google.com/sql/docs/postgres).
+You can however use [Datastream for BigQuery](https://cloud.google.com/datastream/docs) to stream data from PostgreSQL
+hosted anywhere on any cloud.
+
+[Prerequisites](https://cloud.google.com/datastream/docs/before-you-begin) to using DataStream include:
```
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:229]: what feels weird to me is the order of sentences up until this one:
- line 225: these instructions are for Postgres in Cloud SQL
- line 226: but you can use datastream if you use postgres outside of GCP
- line 229: to use datastrea, etc...

Do users also need DataStream for CloudSQL postgres?

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 5]
```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
...
+- creating a Google Cloud account where you create your Google Cloud project
+- enabling billing for the project
+- enabling the Datastream API
+- assigning Datastream Admin role to your user account
+- additional Datastream networking connectivity [requirements](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery#requirements)
+
+You can follow closely the [quickstart Google Cloud documentation](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery) on replicating data from PostgreSQL to BigQuery with Datastream.
```
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:236]:
```suggestion
You can closely follow [this quickstart](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery) on replicating data from PostgreSQL to BigQuery with Datastream.
```

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx (hunk 4 above)
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:229]:
```suggestion
[Prerequisites](https://cloud.google.com/datastream/docs/before-you-begin) to using Datastream include:
```

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 6]
```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
...
+
+The instructions for manual sync include the following steps:
+- Extract data from PostgreSQL to file using `COPY INTO` [command](https://docs.snowflake.com/en/sql-reference/sql/copy-into-location.html).
+  You should also explore the Snowflake [data loading best practices](https://docs.snowflake.com/en/user-guide/data-load-considerations.html) before extraction.
+- Stage the extracted data files to either internal or external locations such as AWS S3, Google Cloud Storage or Microsoft Azure
+- Copy staged files to Snowflake tables using `COPY INTO` [command](https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html).
+  You can decide to use [bulk data loading](https://docs.snowflake.com/en/user-guide/data-load-bulk.html) into Snowflake or to load continuously using [Snowpipe](https://docs.snowflake.com/en/user-guide/data-load-snowpipe.html).
+  Alternatively you can also benefit of [this plugin](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/694157985/Cloud+Storage+to+Snowflake+Action) to load data to an existing Snowflake table.
```
Comment (m-vdb) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:255]:
```suggestion
Alternatively you can also benefit from [this plugin](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/694157985/Cloud+Storage+to+Snowflake+Action) to load data to an existing Snowflake table.
```

File: docs/docs/monitoring/analytics/getting-started-with-analytics.mdx [hunk 7]
```diff
@@ -213,6 +220,36 @@ analytics but lacks the necessary write performance to directly stream data to i
...
 :::
 
+### BigQuery
+
+To stream data from PostgreSQL to BigQuery, you can use [Datastream for BigQuery](https://cloud.google.com/datastream/docs).
+Datastream for BigQuery supports several [PostgreSQL deployment types](https://cloud.google.com/datastream/docs/configure-your-source-postgresql-database#overview),
+including [CloudSQL](https://cloud.google.com/sql/docs/postgres).
+
+Before you begin, make sure to check the Datastream [prerequisites](https://cloud.google.com/datastream/docs/before-you-begin), as well as
+additional Datastream networking connectivity [requirements](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery#requirements)
```
Comment (chandrikas) [inline docs/docs/monitoring/analytics/getting-started-with-analytics.mdx:230]:
```suggestion
additional Datastream networking connectivity [requirements](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery#requirements).
```

**Review by m-vdb — APPROVED** [review]
👍🏻

**Pull request merged**